services/ansible/ansible-quality.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const { floorCount } = require('../../lib/color-formatters')
+const { InvalidResponse } = require('../errors')
+
+const ansibleContentSchema = Joi.object({
+  quality_score: Joi.number()
+    .allow(null)
+    .required(),
+}).required()
+
+class AnsibleGalaxyContent extends BaseJsonService {
+  async fetch({ projectId }) {
+    const url = `https://galaxy.ansible.com/api/v1/content/${projectId}/`
+    return this._requestJson({
+      url,
+      schema: ansibleContentSchema,
+    })
+  }
+}
+
+class AnsibleGalaxyContentQualityScore extends AnsibleGalaxyContent {
+  static render({ qualityScore }) {
+    return {
+      message: qualityScore,
+      color: floorCount(qualityScore, 2, 3, 4),
+    }
+  }
+
+  async handle({ projectId }) {
+    const json = await this.fetch({ projectId })
+
+    const score = json.quality_score
+    console.log(score)
+    if (!score)
```

Comment:
Since this is a falsy check, we will throw the error if the score exists but has a value of 0. I'm not sure what the score scale is, but Is that the desired behavior? is a score of `0` a valid/possibile score? Or does a score of 0 mean that there is no score?